docs/reference/announcements.md

```diff
@@ -24,6 +24,7 @@ Camunda 8.7 is scheduled for release on 8 April 2025.
   - [Fetch RPA resource API](#)
   - [deployResourceAPI for RPA](#) -->
 
+- [Amazon Web Services (AWS) region support](/reference/announcements/870.md#amazon-web-services-aws-region-support-saas)
```

Comment:
I'm wondering if we should add a more generic info about the AWS availability for all SaaS plans?